### DISCUSSION TITLE
What is PD6 function in this?

### DISCUSSION THREAD

**Discussion by xinnie-the-pooh** [xyphro/HP3457-OLED-display#1] [Q&A]
i dont know what does atmega PD6 connect to lm2734 en pin mean. if i use simple circut power ic like ams 1117, it will goes easer to make? and i also cant find any datasheet of this 0.87' lcd ,its vcc should connect to 3.3v? and anther advice, con1 use a smd-like will better and should dont need the second pcb.
in jlc-pcb ,the v-cut is expensive(in CN, it cost 6$USD extra).i also welding circuit by my hand ,0402 is too hard(microscope cant stop my hand shake..),i check drc that also is expensive choice :( the via 12mil looks like its not easy to produce in factory.

**Comment by xyphro** [comment]
Hi Winston,

The oled datasheet mentioned that the power on order of the supply has to
be done in a certain way.

I doubt it is very relevant, but as I used buck converters with enable pin,
this function came for free.

Best regards,

Kai


Winston Smith ***@***.***> schrieb am So., 12. März 2023,
12:06:

**Comment by xyphro** [comment]
Yes the oled vcc is 3.3v. 5v is too much.

You can change the design to your needs. The 2nd pcb was required for good
fit, but if you have another idea or acchept a 5mm distance of the oleds to
the front, you can change it.

The pcbs I used were produced by JLC too.


Winston Smith ***@***.***> schrieb am So., 12. März 2023,
12:06:

**Reply by xinnie-the-pooh** [reply]
luckly ,your issue help me too much. i planing use it to repair my 4263a lcr meter.(i buy it only cost 50usd, but with out psu and lcd,i have repaired the psu done)

i have much 168(arduino mini),it seems could replace atmega8 . also i'm curious that why dont make it by arduino,its more easer to build up. and i think use a uart also make it easy to build up.(maybe will looks ugly)

thanks so much.
![2023_03_13_02_44_IMG_1754](https://user-images.githubusercontent.com/62879756/224565869-7c793c27-328a-4d5c-a48b-ed3c8685797f.JPG)

**Comment by xyphro** [comment]
I did not use Arduino cause I am not so much fan of it, maybe because I
come from a generation that started making code for processors in assembly
and never understood what benefit arduino would add compared to a plain C
project.

Also the i2c libraries of arduino would not have helped, the receiving part
of the display data also has no arduino library, the benefit of arduino
environment would be close to 0.

I also recognize the arduino build environment as extremly slow compared to
normal gcc project and the debugging capabilities of the arduino
environment is limited to just uart prints which impacts timing behaviour a
lot.

All in all it is just a personal preference not to use Arduino. This does
not mean arduino is a bad thing :-)

I released this personal project into the wild to help others and the good
thing is that everybody can adjust it to their needs and own personal
preferences.

Other ppl might prefer micropython, another processor, ... it is hard to
make it right for everybody.



Winston Smith ***@***.***> schrieb am So., 12. März 2023,
19:48:
